### DISCUSSION TITLE
How to apply a magic command in all notebooks by default? The case of %matplotlib inline

### DISCUSSION THREAD

**Discussion by kirilllzaitsev** [microsoft/vscode-jupyter#13395] [Questions and Answers]
Is there any way to avoid copy-pasting `%matplotlib inline` in every notebook and use it by default instead?
What about other magic commands? For tools that don't implement their own .rc configuration file?

**Comment by ccalvocm** [comment]
Greetings @kirilllzaitsev 

You could create any .py file in ~/.ipython/profile_default/startup/ containing

`get_ipython().magic('matplotlib inline')`
